api/client/src/main/java/org/projectnessie/client/http/impl/UriBuilder.java

```diff
@@ -41,12 +41,12 @@ public UriBuilder(URI baseUri) {
   }
 
   public UriBuilder path(String path) {
```

Comment:
The changes in this class are due to the fact that so far, this method was always passed a string that did not contain any slash. Now that this method is used to manipulate user-provided URIs, we need to account for the fact that this method could be called with something like `/foo/bar/qix`.